client/src/app/core/core-services/relation-manager.service.ts

```diff
@@ -132,8 +132,8 @@ export class RelationManagerService {
 
     private handleGenericRelation<M extends BaseModel>(model: M, relation: Relation): any {
         if (relation.many) {
-            const foreignViewModels = model[relation.ownIdField].map(fqid => {
-                let collection, id;
+            const foreignViewModels = (model[relation.ownIdField] || []).map((fqid: string) => {
+                let collection: string, id: number;
```

Comment:
If you use typedefs, you can use `Fqid`, `Collection`, `Id` ;)

Reply:
Right :D